test/test_install.sh

```
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+
+install_script="$repo_root/install.sh"
```

Shellcheck (0.11.0)

[warning] repo_root is referenced but not assigned.

(SC2154)